### PR TITLE
add using questions in json format

### PR DESCRIPTION
**Pull request by AlexaZhuk**

### CODE REVIEW

File: QuizApp/Program.cs [hunk 1]
```diff
@@ -1,63 +1,40 @@
...
-			{
-				taxt[j] = taxt[j].TrimEnd('*');
-			}
-			NumberQ(taxt);
+			string json = File.ReadAllText("C:\\Users\\Alexa\\Desktop\\test\\q.json");
+			AllQuestions info = JsonConvert.DeserializeObject<AllQuestions>(json);
 
-		}
+			int count = 0;
+			int countAmswer = 1;
```
Comment (MrSantiaga) [inline QuizApp/Program.cs]: typo

File: QuizApp/Program.cs [hunk 2]
```diff
@@ -1,63 +1,40 @@
...
-			string path = "C:\\Users\\Alexa\\Desktop\\test\\1.txt";
-			string[] taxt = File.ReadAllLines(path);
 
-			for (int j = 0; j < taxt.Length; j++)
-			{
-				taxt[j] = taxt[j].TrimEnd('*');
-			}
-			NumberQ(taxt);
+			string json = File.ReadAllText("C:\\Users\\Alexa\\Desktop\\test\\q.json");
+			AllQuestions info = JsonConvert.DeserializeObject<AllQuestions>(json);
```
Comment (MrSantiaga) [inline QuizApp/Program.cs]: it is not info, int is, e.g., questionsData

File: QuizApp/Program.cs [hunk 3]
```diff
@@ -1,63 +1,40 @@
...
+			int countAmswer = 1;
 
-		static void NumberQ(string[] taxt)
-		{
-			int result;
-			int point = 0;
-			int count;
-
-			for (int i = 1; i < 11; i++)
+			foreach (var quDescription in info.Questions)
```
Comment (MrSantiaga) [inline QuizApp/Program.cs]: dont use var

File: QuizApp/Program.cs [hunk 4]
```diff
@@ -1,63 +1,40 @@
...
 			{
-				for (int k = (i - 1) * 6; k < i * 6 && k < taxt.Length; k++)
-				{
-					Console.WriteLine(taxt[k]);
-				}
-				Console.WriteLine("Choose the answer: ");
+				Console.WriteLine(quDescription.Question);
+				foreach (var item in quDescription.Answers)
```
Comment (MrSantiaga) [inline QuizApp/Program.cs]: var

File: QuizApp/Program.cs (hunk 3 above)
Comment (MrSantiaga) [inline QuizApp/Program.cs]: just description

File: QuizApp/Program.cs [hunk 5]
```diff
@@ -1,63 +1,40 @@
...
-				string path = "C:\\Users\\Alexa\\Desktop\\test\\1.txt";
-				string[] taxtFifst = File.ReadAllLines(path);
 
-				if (i == 1)
-				{
-					count = result;
-				}
-				else
+				int userAnswer;
+
```
Comment (MrSantiaga) [inline QuizApp/Program.cs]: extra line

File: QuizApp/Program.cs (hunk 1 above)
Comment (MrSantiaga) [inline QuizApp/Program.cs]: close to usage approach

**Review by MrSantiaga — CHANGES_REQUESTED** [review]

File: QuizApp/Program.cs (hunk 1 above)
Comment (AlexaZhuk) [reply QuizApp/Program.cs]: its too much

File: QuizApp/Program.cs (hunk 2 above)
Comment (AlexaZhuk) [reply QuizApp/Program.cs]: ohhh

File: QuizApp/Program.cs (hunk 3 above)
Comment (AlexaZhuk) [reply QuizApp/Program.cs]: be be be

File: QuizApp/Program.cs [hunk 6]
```diff
@@ -1,63 +1,39 @@
...
+			foreach (QuestionDescription description in questionsData.Questions)
 			{
-				for (int k = (i - 1) * 6; k < i * 6 && k < taxt.Length; k++)
-				{
-					Console.WriteLine(taxt[k]);
-				}
-				Console.WriteLine("Choose the answer: ");
+				int countAnswer = 1;
+				Console.WriteLine(description.Question);
+				foreach (string item in description.Answers)
```
Comment (MrSantiaga) [inline QuizApp/Program.cs]: extra line

File: QuizApp/Program.cs [hunk 7]
```diff
@@ -1,63 +1,39 @@
...
+﻿using Newtonsoft.Json;
+
+namespace QuizApp
 {
 	internal class Program
 	{
 		static void Main(string[] args)
 		{
-			string path = "C:\\Users\\Alexa\\Desktop\\test\\1.txt";
-			string[] taxt = File.ReadAllLines(path);
```
Comment (MrSantiaga) [inline QuizApp/Program.cs]: extra line

File: QuizApp/Program.cs [hunk 8]
```diff
@@ -1,63 +1,39 @@
...
-			for (int j = 0; j < taxt.Length; j++)
-			{
-				taxt[j] = taxt[j].TrimEnd('*');
-			}
-			NumberQ(taxt);
+			string json = File.ReadAllText("C:\\Users\\Alexa\\Desktop\\test\\q.json");
+			AllQuestions questionsData = JsonConvert.DeserializeObject<AllQuestions>(json);
 
-		}
+			int count = 0;
```
Comment (MrSantiaga) [inline QuizApp/Program.cs]: extra line

File: QuizApp/Program.cs [hunk 9]
```diff
@@ -1,63 +1,39 @@
...
 
-				if (answer.EndsWith('*'))
+				if (userAnswer == description.RightIndex + 1)
 				{
-					point += 1;
+					count += 1;
 				}
 			}
-			Console.WriteLine($"your points = {point}");
+			Console.WriteLine($"You have {count} right answers from {questionsData.Questions.Count} \n");
```
Comment (MrSantiaga) [inline QuizApp/Program.cs]: add line before `Console.WriteLine`

**Review by MrSantiaga — CHANGES_REQUESTED** [review]

File: QuizApp/Program.cs [hunk 10]
```diff
@@ -1,63 +1,37 @@
...
-		static void NumberQ(string[] taxt)
-		{
-			int result;
-			int point = 0;
-			int count;
-
-			for (int i = 1; i < 11; i++)
-			{
-				for (int k = (i - 1) * 6; k < i * 6 && k < taxt.Length; k++)
+				int countAnswer = 1;
```
Comment (MrSantiaga) [inline QuizApp/Program.cs]: countAnswer -> answerNumber

File: QuizApp/Program.cs [hunk 11]
```diff
@@ -1,63 +1,37 @@
...
+				Console.WriteLine(description.Question);
+				foreach (string item in description.Answers)
```
Comment (MrSantiaga) [inline QuizApp/Program.cs]: item -> answer

File: QuizApp/Program.cs [hunk 12]
```diff
@@ -1,63 +1,37 @@
...
 	{
 		static void Main(string[] args)
 		{
-			string path = "C:\\Users\\Alexa\\Desktop\\test\\1.txt";
-			string[] taxt = File.ReadAllLines(path);
+			string json = File.ReadAllText("C:\\Users\\Alexa\\Desktop\\test\\q.json");
+			AllQuestions questionsData = JsonConvert.DeserializeObject<AllQuestions>(json);
 
-			for (int j = 0; j < taxt.Length; j++)
+			int count = 0;
```
Comment (MrSantiaga) [inline QuizApp/Program.cs]: count of what?

**Review by MrSantiaga — CHANGES_REQUESTED** [review]

**Review by MrSantiaga — APPROVED** [review]

**Pull request merged**